src/vector/_methods.py

```diff
@@ -194,140 +196,140 @@ def to_Vector4D(self) -> "VectorProtocolLorentz":
         """
         raise AssertionError
 
-    def to_xy(self) -> "VectorProtocolPlanar":
```

Comment:
I am not sure how all the quotes from the type hints went away. Was this a pre-commit hook?

Reply:
Yes, PyUpgrade will do this if `from __future__ import annotations` is present.